### PR TITLE
Add "last_changed" and "last_updated" to dev tools state view

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Currently there is no easy way to see those timestamps. Now there is. I decided against formatting the values and instead just added the default ISO format since we are talking about dev tools.

The filter for the state column still only looks at the actual state value, so no negative impact here.

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
Not to sure about this, it makes it a lot harder to scan for states

**Comment by spacegaier** [comment]
Initially I had though about adding another column for those timestamps, but since most entities anyway have multiple lines of attributes, there was always unused space in the state column which felt better.

Reg. scanning of states: If you are looking for something specific you would anyway use the column filters, wouldn't you?

**Comment by bramkragten** [comment]
What if I don't know what I'm looking for, and want to see the differences

**Comment by spacegaier** [comment]
Even then I think it is not "too bad" since the timestamps in term of visual outline are always the same size, so I feel scanning through still works reasonably well. But I get your overall point...

So options are:
1. New column in table (something like "State Updates")
2. Line break after state before the timestamp to have the state stand out a bit more
3. Have the timestamp not use the main text color, but something grayer (although that might also look weird)

**Comment by bramkragten** [comment]
Try using `secondary-text-color` and see how that looks.

**Comment by spacegaier** [comment]
I like it esp. combined with an extra line break.
image

File: src/panels/developer-tools/state/developer-tools-state.js [hunk 1]
```diff
@@ -168,7 +175,15 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
...
             </td>
-            <td>[[entity.state]]</td>
+            <td>
+              [[entity.state]]<br /><br />
+              <span class="secondary"
+                >last_changed: [[lastChangedString(entity)]]</span
+              ><br />
+              <span class="secondary"
+                >last_updated: [[lastUpdatedString(entity)]]</span
+              >
```
Comment (bramkragten) [inline src/panels/developer-tools/state/developer-tools-state.js:185]:
```suggestion
              <span class="secondary"
                >last_changed: [[lastChangedString(entity)]]<br />
                last_updated: [[lastUpdatedString(entity)]]</span
              >
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**